app/templates/client_ng1/dev/index.html

```diff
@@ -1,35 +1,43 @@
 <!doctype html>
 <html ng-app="myAwesomeApp">
-	<head>
-		<meta charset="utf-8">
+  <head>
     <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
+    <meta name="description" content="My awesome application">
+    <meta name="keywords" content="Todo,myAwesomeApp,App">
+    <meta name="author" content="Author">
     <title>todo :D</title>
-    <link rel="shortcut icon" type="image/ico" href="favicon.png" />
-    <base href="/" />
-
-    <link type="text/css" rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.min.css?rev=@@hash" />
-    <link type="text/css" rel="stylesheet" href="bower_components/fontawesome/css/font-awesome.min.css?rev=@@hash" />
-
-    <link type="text/css" rel="stylesheet" href="todo/styles/frameworks_overrides.css?rev=@@hash" />
-    <link type="text/css" rel="stylesheet" href="todo/styles/fonts.css?rev=@@hash" />
-    <link type="text/css" rel="stylesheet" href="todo/styles/position.css?rev=@@hash" />
-    <link type="text/css" rel="stylesheet" href="todo/styles/styles.css?rev=@@hash" />
-    <link type="text/css" rel="stylesheet" href="todo/styles/media_queries.css?rev=@@hash" />
-    <link type="text/css" rel="stylesheet" href="todo/styles/events.css?rev=@@hash" />
-
-	</head>
-
-	<body>
-
-		<div ng-view></div>
-
+    <link rel="shortcut icon" type="image/ico" href="favicon.png"/>
+    <base href="/"/>
+
```

Comment:
`<base href="/" />` has to be the first element, otherwise some routes might fail.